src/init.cpp

```diff
@@ -932,10 +932,6 @@ bool AppInitParameterInteraction(const ArgsManager& args, bool use_syscall_sandb
     }
 
     // mempool limits
```

Comment:
7e90c69f1be6f8ba52e1f24b983379d00aad2f65 : forgot to remove this?

Reply:
Fixed as of a956afedfe